src/app/services/solrSrv.js

```diff
@@ -62,6 +62,29 @@ function (angular, _) {
         //     }
         // });
     };
+
+    // Send user review to Solr
+    this.postReview = function(review) {
+        jQuery.ajax({
+        type: "POST",
+        data: JSON.stringify(review),
+        contentType: "application/json; charset=utf-8",
+        dataType: "json",
+        url: dashboard.current.solr.server + "user/accuracy-review",
+        crossDomain: true,
+        beforeSend: function(xhr){
+            xhr.withCredentials = true;
+        },
+        success: function(data, textStatus, request){
+            console.log(data, textStatus, request)
+            alert("your review has been submitted");
+        },
+        error: function(data, textStatus, request){
+            console.log(data, textStatus, request)
+            alert('POST request error!');
```

Comment:
Maybe for end-users a more user-friendly message would be "Failed to send your review to the server". Are you using the Banana alert service (probably best to use this), or the browser alert?

Reply:
Changed the message and replaced the browser alert with the Banana ones; However, the buttons to send the reviews are hide at the moment since we are changing the _table_ with the _cards_.